index.html

```diff
@@ -947,620 +974,1301 @@
         </div>
       </div>
 
-      <div class="container d-lg-none">
+      <div class="container d-none d-lg-block main2 mt-5 pt-5">
         <div class="row">
-          <div class="col-12 hidden mit">
-            <div class="featured">TOP STORY</div>
-            <div class="relative-img">
-              <img src="./assets/mitt-romney.jpg" alt="mit" />
-              <div class="square rom">U.S</div>
+          <div class="col-lg-12 in-magazine">
+            <div class="row in-magazine-title">
+              <div
+                class="col-lg-12 d-lg-flex border-bottom pb-1 border-secondary justify-content-center"
+              >
+                <img
+                  class="img-fluid n-logo-small"
+                  src="./assets/n-logo-small.JPG"
+                  alt="Newsweek logo small"
+                />
+                <strong class="ml-2">IN THE MAGAZINE</strong>
+              </div>
             </div>
+            <div class="row in-magazine-row1">
+              <div class="col-lg-3 pl-0 d-flex flex-column magazine-front-page">
+                <div class="relative-img pt-2">
+                  <img
+                    class="img-fluid img-magazine-height-desktop"
+                    src="./assets/in-magazine-cover.JPG"
+                    alt="cover of magazine"
+                  />
+                  <div class="square-magazine">JANUARY 22 ISSUE</div>
+                </div>
+                <strong class="px-auto pl-5 magazine-all-features"
+                  >SEE ALL FEATURES</strong
+                >
+              </div>

```

Comment:
- [ ] Every article inside a section in the Newsweek website has a link around the image of the article and another link also in the title/text of the article.
Make sure to use `a` tags around the images and titles of every article in your clone, including from previous milestones.

- [ ] Make sure that each title for each article is also using a header tag, h2, h3, h4, etc.